janusgraph-test/src/test/java/org/janusgraph/graphdb/management/ConfigurationManagementGraphTest.java

```diff
@@ -27,16 +28,34 @@
 import static org.janusgraph.graphdb.configuration.GraphDatabaseConfiguration.STORAGE_BACKEND;
 
 import org.apache.tinkerpop.gremlin.server.Settings;
+import org.apache.tinkerpop.gremlin.structure.Vertex;
 import org.apache.commons.configuration.MapConfiguration;
 
+import java.lang.reflect.Field;
 import java.util.Map;
 import java.util.HashMap;
 
+import org.junit.After;
 import org.junit.Test;
 import static org.junit.Assert.*;
 
 public class ConfigurationManagementGraphTest {
 
+    @After
+    public void removeStaticSingletonAftetTest() {
+        try {
+            final Field gmInstance = JanusGraphManager.class.getDeclaredField("instance");
+            gmInstance.setAccessible(true);
+            gmInstance.set(null, null);
+
+            final Field cmgInstance = ConfigurationManagementGraph.class.getDeclaredField("instance");
+            cmgInstance.setAccessible(true);
+            cmgInstance.set(null, null);
+        } catch (NoSuchFieldException | IllegalAccessException e) {
+            throw new RuntimeException(e);
+        }
+    }
+
```

Comment:
Nice work.
Only suggestion I have is that JanusGraphManager actually has a method that already does this named `shutdownJanusGraphManager()`

If you added a similiar method to `/janusgraph-core/src/main/java/org/janusgraph/graphdb/management/ConfigurationManagementGraph.java`
like:
``` 
    // To be used for testing purposes
    protected static void shutdownConfigurationManagementGraph() {
        instance = null;
    }
```

you could simplify the try block to 
```
JanusGraphManager.shutdownJanusGraphManager();
ConfigurationManagementGraph.shutdownConfigurationManagementGraph();
```

How's it going with the CLA?

Reply:
Done
CLA was sent on Jan 29